server/models/Event.js

```diff
@@ -13,9 +13,14 @@ const eventSchema = new mongoose.Schema({
     type: Date,
     required: true
   },
-  // eventTimeZone:{
-  //   type: Date,
-  // }
+  eventLink:{
+    type:String,
+    required: true
+  },
```

Comment:
Storing a link to every event type might be problematic if we allow a user to edit event type durations. What if a user bookmarks that url on their browser, then that would no longer work. 

I think we can make this a bit more dynamic in our routing.

Reply:
Yeah, I will take the eventLink field out of the module